source/linux/How_to_Guides/Target/How_to_manually_flash_emmc_device.rst

```diff
@@ -0,0 +1,217 @@
+How to flash eMMC device
+========================
```

Comment:
This is a new file that is not being included by any other file, start with
```rst
########################
How to flash eMMC device
########################
```
And continue following https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
Ahh... little bit confused about the headings but will take a look

Reply:
You got it. The gist is, section ordinals are only respected in the current file being parsed. They can and usually should always start at the highest ordinal to make it clear what section depth you're in.

The only exception is included files. Unless you pass the included files through the parser early with a special directive, they are basically read and injected into the current file wherever the include directive is. Because of this, the section depth actually has to be watched a little more closely, as it can result in something like this:

file 1 structure:
```
Part A
├── Chapter B
include 2
└── Chapter D
```

file 2 structure:
```
Part C
├── Chapter C.1
└── Chapter C.2
```

At first glance it looks like this is fine. A is at ordinal 0 and D is at ordinal 1, it should fall under A, but when C is included it actually falls under a new Part because C begins at ordinal 0 as well.

```
A
└── B
C
├── C.1
├── C.2
└── D 
```

This is an issue with include directives in general, and part of the reason I try to prevent people from using them without good reason. It isn't affected by what structure we use, but keeping the ordinals high makes it easier to spot.

Reply:
Ahh okay, thanks for the explanation!